pkg/api/admin/openshiftcluster.go

```diff
@@ -28,7 +28,7 @@ type OpenShiftCluster struct {
 
 // OpenShiftClusterProperties represents an OpenShift cluster's properties.
 type OpenShiftClusterProperties struct {
-	ArchitectureVersion     ArchitectureVersion     `json:"architectureVersion,omitempty"`
+	ArchitectureVersion     ArchitectureVersion     `json:"architectureVersion"` // ArchitectureVersion is int so 0 is valid value to be returned
```

Comment:
>  // ArchitectureVersion is int so 0 is valid value to be returned

To confirm my understanding, `0` is `ArchitectureVersion1` and `1` is `ArchitectureVersion2`?

Reply:
yes :) we needed default for all clusters without need to go and fix all of them :)

Reply:
Can we skip ArchitectureVersion1 and go straight from 0 to 2 or realign the numbers and names somehow? It's easier to make a mistake with this kind of misalignment.

Reply:
Dont think so :/

Reply:
If we really want to realign the int value with the version, we can try setting `ArchitectureVersionV1 ArchitectureVersion = iota + 1 [here](https://github.com/Azure/ARO-RP/blob/00f28f958ab5e2024f2478ef5445087023a0db5f/pkg/api/admin/openshiftcluster.go#L181). I want to say it looks like a safe change, but not :100: sure.